change/@uifabric-api-docs-2019-12-27-17-16-21-jg-build-and-test.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
Shouldn't most/all of these be `none` since there's no impact to the published packages?